packages/test/src/test/history-test.ts

```diff
@@ -60,6 +61,60 @@ describe("Repository tests", () => {
         })
     })
 
+    describe("Partitions CRUD", () => {
+        it("test multiple ast changes", async () => {
+
+            function simpleNode(id: string): LionWebJsonNode {
+                return {
+                    id,
+                    classifier: {
+                        language: "my-language",
+                        key: "my-concept",
+                        version: "1"
+                    },
+                    properties: [],
+                    containments: [],
+                    references: [],
+                    annotations: [],
+                    parent: null
+                }
+            }
+
+            function getRepoVersion(response: ClientResponse<ListPartitionsResponse|CreatePartitionsResponse>) : number {
+                assert(response.body.success);
+                return Number.parseInt(response.body.messages.find(m => m.kind === "RepoVersion")?.data?.version)
+            }
+
+            function idsInChunk(response: ClientResponse<ListPartitionsResponse>) : Set<string> {
+                assert(response.body.success);
+                return new Set<string>(response.body.chunk.nodes.map(n => n.id));
+            }
+
+            const client = new RepositoryClient("TestHistoryClient", "history-partition-crud");
+            client.dbAdmin.createRepository("history-partition-crud", true, "2023.1");
+            const v1 = getRepoVersion(await client.bulk.createPartitions({
+                languages: [
+                    {
+                        key: "my-language",
+                        version: "1"
+                    }
+                ], nodes: [
+                    simpleNode("id-abc"),
+                    simpleNode("id-bcd"),
+                    simpleNode("id-cde")
+                ], serializationFormatVersion: "2023.1"
+            }));
+            const v2 = getRepoVersion(await client.bulk.deletePartitions(["id-abc"]));
+            const v3 = getRepoVersion(await client.bulk.deletePartitions(["id-bcd"]));
+            const v4 = getRepoVersion(await client.bulk.deletePartitions(["id-cde"]));
+            expect(idsInChunk(await client.history.listPartitions(0))).to.deep.equal(new Set<string>([]));
```

Comment:
I added asserts to test whether the version number after each operation is the previous + 1.